services/chocolatey/chocolatey.service.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const { createServiceFamily } = require('../nuget/nuget-v2-service-family')
+
+module.exports = createServiceFamily({
```

Comment:
I know you've just left the examples in `all-badge-examples` for now, but is the plan here to override `examples()` in these files so we do something like:

```js
{ Downloads, Version } = createServiceFamily({ /*...*/ })

Downloads.examples = function() { /*...*/ }

Version.examples = function() { /*...*/ }

module.exports = { Downloads, Version }
```

(not tested, but you get the idea)

Reply:
Yea, that's a possibility. The other possibility is that we can modify the examples to use the same example across all the badges for a given family. That way they'll be easier to generate programmatically. That approach probably won't work for myget, though, which has a lot of variants.